packages/backend/src/types/schemaHelpers.ts

```diff
@@ -0,0 +1,146 @@
+import { PendingRating, Professor, Rating, TruncatedProfessor } from "./schema";
+
```

Comment:
Have more of a high-level comment that this file should probably be called `adapters.ts`